src/main/java/com/google/api/codegen/py/PythonProtoFileInitializer.java

```diff
@@ -14,14 +14,23 @@
  */
 package com.google.api.codegen.py;
 
+import com.google.api.codegen.ProtoFileView;
 import com.google.api.tools.framework.model.ProtoFile;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Sets;
+import java.util.Set;
 
 public class PythonProtoFileInitializer implements PythonSnippetSetInputInitializer<ProtoFile> {
 
+  private Set<ProtoFile> importableProtoFiles = null;
```

Comment:
If we did, we would traverse the entire model every for every proto file we create a dummy of. This way we only do the computation once.